src/index.html

```diff
@@ -5,8 +5,482 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          src="./images/logo.png"
+          alt="Logo Moyo"
+        >
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="#" class="header__item-link is-active">
```

Comment:
it would be great to add specific links for every `a` (e.g. `#apple`,` #samsung` etc.)
```suggestion
            <a href="#apple" class="header__item-link is-active">
```

Reply:
fix